modules/demo_account/scripts/sample_deploys.sh

```diff
@@ -1,14 +1,25 @@
 #! /bin/bash
 
 deploy() {
-  curl -s -X POST $1 \
+  # Detect if we are on macOS or Linux
+  if [[ "$OSTYPE" == "darwin"* ]]; then
+    # macOS date command
+    formatted_date=$(date -u $3 '+%FT%TZ')
+  else
+    # Linux date command
+    # Replace `-v` flag and format the input for Linux `date`
+    adjusted=$(echo $3 | sed 's/-v //g' | sed 's/H/hour/g; s/d/day/g')
+    formatted_date=$(date -u -d "$adjusted" '+%FT%TZ')
```

Comment:
i'm going out on a limb with chatGPT here

Reply:
On Ubuntu the output is something like
`2024-12-23T19:42:16Z` when `$3` is `"-v -1H"`